spec/features/item_creation_no_files_spec.rb

```diff
@@ -49,8 +49,11 @@
     expect(page).to have_text("Project : #{project}")
     expect(page).to have_text("Registered By : #{AuthenticationHelpers.username}")
 
-    # wait for accessioningWF to finish
-    reload_page_until_timeout!(text: 'v1 Accessioned')
+    # wait for accessioningWF to finish; retry if Version mismatch on sdr-ingest-transfer
+    reload_page_until_timeout_with_wf_step_retry!(expected_text: 'v1 Accessioned',
+                                                  workflow: 'accessionWF',
+                                                  workflow_retry_text: 'Version mismatch',
+                                                  retry_wait: 2)
```

Comment:
the addition of the retry here and below made this test pass much more consistently.